versions.lock

```diff
@@ -73,44 +75,45 @@ javax.annotation:javax.annotation-api:1.3.2 (3 constraints: 55341c48)
 javax.inject:javax.inject:1 (4 constraints: 852d0c1a)
 javax.jdo:jdo-api:3.0.1 (2 constraints: 4c1dcc1a)
 javax.servlet:javax.servlet-api:3.1.0 (1 constraints: 150dc436)
-javax.servlet:servlet-api:2.5 (7 constraints: c87e5aa1)
+javax.servlet:jsp-api:2.0 (1 constraints: 0b0aa0a7)
+javax.servlet:servlet-api:2.5 (9 constraints: f991a6d2)
 javax.servlet.jsp:jsp-api:2.1 (1 constraints: 290d5a14)
 javax.transaction:jta:1.1 (1 constraints: 9f07d96b)
+javax.transaction:transaction-api:1.1 (1 constraints: 0a0b64c9)
```

Comment:
Is this needed?

Reply:
Introduced by datanucleus, seems not necessary, I will remove it.

```
\--- org.datanucleus:javax.jdo:3.2.0-m3
     \--- org.apache.hive:hive-metastore:2.3.6
```